src/OnboardingSPA/components/GridView/stylesheet.scss

```diff
@@ -0,0 +1,29 @@
+.grid-basic {
```

Comment:
Should we add an onboarding prefix to the CSS selectors? This will ensure that it applies only this component, given that `grid` is a very common name.

Reply:
Sure, Thanks